app/src/main/java/org/wikipedia/onboarding/OnboardingPageView.kt

```diff
@@ -81,6 +83,15 @@ class OnboardingPageView constructor(context: Context, attrs: AttributeSet? = nu
         }
     }
 
+    fun removeMAinTextLayoutGravity() {
```

Comment:
There are two upper cases in the function name.

Reply:
Done @cooltey